app/src/main/java/com/brandon/forzenbook/usecase/GetLoginCodeUseCaseImpl.kt

```diff
@@ -4,15 +4,17 @@ import com.brandon.forzenbook.repository.ForzenRepository
 
 class GetLoginCodeUseCaseImpl(
     private val forzenRepository: ForzenRepository
-): GetLoginCodeUseCase {
+) : GetLoginCodeUseCase, AccountValidation() {
```

Comment:
We can just call this `LoginUseCase`, the code implementation detail doesn't need to be part of the name

Reply:
I see that you already have a login use case, just make sure namings don't include code as a detail

Reply:
renamed use case and combined it with another one since they did the same type of work.